src/jbi/whiteboard_actions/default.py

```diff
@@ -36,15 +36,13 @@ def __init__(self, whiteboard_tag, jira_project_key, **kwargs):
         self.jira_client = get_jira()
 
     def __call__(  # pylint: disable=inconsistent-return-statements
-        self, payload: BugzillaWebhookRequest
+        self, payload: BugzillaWebhookRequest, bug_obj: BugzillaBug
```

Comment:
I'm not sure whether it is cleaner than your proposal, but alternatively, to avoid changing actions signatures, we could add a property to the `BugzillaWebhookRequest` class:

```py
@functools.cached_property
  def bugzilla_object(self):
    if not self.bug:
         raise ValueError("boom")
    return getbug_as_bugzilla_object(self)
```

Reply:
That's a good idea, I'll do that.